fuel-client/assets/schema.sdl

```diff
@@ -65,6 +65,10 @@ type BlockEdge {
 	cursor: String!
 }
 scalar BlockId
+input Breakpoint {
+	contract: [Int!]!
```

Comment:
Shouldn't this be a contract id? I wonder why its the same type of `$pc`, Is it an index of an internal mapping? If yes, then we might rather use the contract ID instead of relying in the debugger state in order to interact with it